erpnext/setup/doctype/company/test_company_production.js

```diff
@@ -0,0 +1,19 @@
+QUnit.test("Test: Company", function (assert) {
```

Comment:
if you are writing multiple UI tests for a doctype, add it in a `tests` folder inside the doctype